SwitchAudioSource.py

```diff
@@ -33,15 +33,21 @@ def get_sources():
     ]).strip()
 
     command_output = check_output([
-        PATH_TO_SWITCH_AUDIO_OUTPUT, '-a', '-t', 'output'
```

Comment:
This `-t output` seems extraneous. It's filtered again by the python script that calls this method.

Reply:
Good catch!